api/app/signals/apps/signals/urls.py

```diff
@@ -17,55 +15,12 @@
 signal_router_v0.register(r'auth/priority', v0_views.PriorityAuthViewSet, base_name='priority-auth')
 signal_router_v0.register(r'auth/note', v0_views.NoteAuthViewSet, base_name='note-auth')
 
-# API Version 1
-signal_router_v1 = SignalsRouterVersion1()
-signal_router_v1.register(r'public/terms/categories',
-                          v1_views.MainCategoryViewSet,
-                          base_name='category')
-signal_router_v1.register(r'private/signals',
-                          v1_views.PrivateSignalViewSet,
-                          base_name='private-signals')
-signal_router_v1.register(r'public/signals',
-                          v1_views.PublicSignalViewSet,
-                          base_name='public-signals')
-
-signal_router_v1.urls.append(
-    path('private/signals/<int:pk>/split',
-         v1_views.PrivateSignalSplitViewSet.as_view({'get': 'retrieve', 'post': 'create'}),
-         name='private-signals-split')
-)
-
-signal_router_v1.urls.append(
-    path('private/signals/<int:pk>/attachments',
-         v1_views.PrivateSignalAttachmentsViewSet.as_view({'get': 'list', 'post': 'create'}),
-         name='private-signals-attachments')
-)
-
-signal_router_v1.urls.append(
-    path('public/signals/<str:signal_id>/attachments',
-         v1_views.PublicSignalAttachmentsViewSet.as_view({'post': 'create'}),
-         name='public-signals-attachments')
-)
-
-# Appending extra url route for sub category detail endpoint.
-signal_router_v1.urls.append(
-    path('public/terms/categories/<str:slug>/sub_categories/<str:sub_slug>',
-         v1_views.SubCategoryViewSet.as_view({'get': 'retrieve'}),
-         name='sub-category-detail'),
-)
-
-signal_router_v1.urls.append(
-    path('pdf/SIA-<int:signal_id>.pdf',
-         v1_views.GeneratePdfView.as_view(),
-         name='signal-pdf-download'),
-)
-
 urlpatterns = [
-    # API Version 0
+    # AP Version 0
     path('', include((signal_router_v0.urls, 'signals'), namespace='v0')),
 
     # API Version 1
-    path('v1/', include((signal_router_v1.urls, 'signals'), namespace='v1')),
+    path('', include(('signals.apps.signals.v1.urls', 'signals'), namespace='v1')),
```

Comment:
Nice, is more explicit --- but for symmetry change the one above as well (the V0 one).

Reply:
Didn't want to touch the V0 to much in this MR. Let's pick this up in a new MR.